src/components/Database/DataViewer/NodeLink.tsx

```diff
@@ -27,7 +27,7 @@ export interface Props {
 }
 
 export const NodeLink = React.memo<Props>(function NodeLink$({ dbRef }) {
-  const key = dbRef.parent === null ? getDbRootUrl(dbRef) : dbRef.key;
+  const key = dbRef.key || getDbRootUrl(dbRef);
```

Comment:
Does RTDB allow a key that's the empty string?

Reply:
Good Q, I don't think so. but let's check

Reply:
Nope, phew!

```
database.ref().child('///a//').key
// "a"

database.ref().child('///a//').toString()
"http://localhost:9000/a"
```